### PR TITLE
#151632815 Add a shell command

### PR DESCRIPTION
**Pull request by andela-Taiwo**
#### What does this PR do?
- Add shell command that makes it easier to perform query operations on the tables in command line
#### Description of Task to be completed?
- create a shell command 
#### How should this be manually tested?
 - ```python manage.py shell```
#### What are the relevant pivotal tracker stories?
-  #151632815

#### Screenshots (if appropriate)

screen shot 2017-10-17 at 7 03 30 pm
screen shot 2017-10-17 at 7 06 35 pm

screen shot 2017-10-17 at 7 07 07 pm

#### Questions:
- None

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -20,6 +20,15 @@
 # initialize flask script
 manager = Manager(app)
 
+
+def shell_command():
```
Comment (georgreen) [inline manage.py]: Add a doc string here

Comment (andela-Taiwo) [reply manage.py]: Noted

File: manage.py [hunk 2]
```diff
@@ -20,6 +20,24 @@
...
+    Args:
+        None
+
+    Returns:
+      (dict) dictionary that include the specified models and database
+
+    """
+    return dict(ContributionType=ContributionType, Contribution=Contribution, Tag=Tag, Comment=Comment, Notification=Notification, User=User, Vote=Vote,
+    contribution_tag=contribution_tag, Bookmark=Bookmark, Path=Path, db=db, Level=Level, path_level=path_level, path_level_skill=path_level_skill, Skill=Skill)
+
```
Comment (wanjiku-carol) [inline manage.py]: Add `Role`, its the newest model. It hasn't been imported in this file.

Comment (andela-Taiwo) [reply manage.py]: Okay

**Comment by wanjiku-carol** [comment]
LGTM

**Comment by andela-milesanmi** [comment]
LGTM

**Comment by andela-ieyo** [comment]
LGTM

**Pull request merged**